.rubocop_todo.yml

```diff
@@ -0,0 +1,45 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2023-03-24 14:42:11 UTC using RuboCop version 1.28.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: IgnoredMethods, CountRepeatedAttributes.
+Metrics/AbcSize:
```

Comment:
I'm tempted to disable the metrics cops.

Reply:
will do so in a follow up PR to introduce voxpupuli-rubocop